python2-pybind11/setup.py

```diff
@@ -3,7 +3,7 @@
 from distutils.core import setup, Extension
 from distutils import sysconfig
 
-cpp_args = ['-std=c++14', '-stdlib=libc++', '-mmacosx-version-min=10.7']
```

Comment:
See this for the recommended way of setting up compiler flags: https://github.com/pybind/pbtest/blob/master/setup.py